data_pipeline_api/java_api/src/test/resources/config.yaml

```diff
@@ -0,0 +1,23 @@
+data_directory: data
+access_log: access-{run_id}.yaml
```

Comment:
This access_log line looks odd, but I guess it doesn't make any difference to anything.

Reply:
@johnnonweiler Copied this from the original python implementation. Seems that in the python implementation they replace `{run_id}` with the actual run id. @rtblackwell @mrow84 is that correct?

Reply:
Yes, there is a description of the mechanism in the [file API spec](https://teams.microsoft.com/l/file/8B530C1E-25FE-4896-BD0C-8FC8089C5AC7?tenantId=6e725c29-763a-4f50-81f2-2e254f0133c8&fileType=docx&objectUrl=https%3A%2F%2Fgla.sharepoint.com%2Fsites%2FScottishCOVID-19ResearchConsortium%2FShared%20Documents%2FGeneral%2FCollaboration%2Fdata%20pipeline%20API%2F1.%20File%20API.docx&baseUrl=https%3A%2F%2Fgla.sharepoint.com%2Fsites%2FScottishCOVID-19ResearchConsortium&serviceName=teams&threadId=19:283c5ea4551344249cc76dd0ecaa4f74@thread.tacv2&groupId=669b45e2-a9a6-4060-ba8a-9ebd3713e367) (link may or may not work, file is in teams at: General > Collaboration > data pipeline API).

Reply:
Might there be any other instances of {run_id} in the yaml file that should be replaced, or is this just for the access_log?

Reply:
I believe it's an edge case specific to the access_log. @mrow84 can you confirm if this is true? Thanks

Reply:
The underlying feature is to associate output files with the run that produced them. There are two categories of output file: those produced by the model, handled by the file API write mechanism, and the access log. Both categories are identified using the run id - the former during the `open_for_write` process, and the latter as this transformation of the input filename. Ideally you _always_ want the run id in the access log filename, but we allow for the case where that isn't so (which can be useful).

I don't think the transformation is required elsewhere.

Reply:
Thanks Chris.